### PR TITLE
Remove whitespace in shortcakes without atts

### PR DESCRIPTION
**Pull request by Kuzmanov**
Using the attrs array length to see if attributes should or shouldn’t be added in the shortcode.
#334

### CODE REVIEW

**Comment by danielbachhuber** [comment]
@Kuzmanov Thanks for the PR! Can you merge master to fix the build error?

File: js/build/shortcode-ui.js [hunk 1]
```diff
@@ -612,6 +618,13 @@ var editAttributeField = Backbone.View.extend( {
 			id: 'shortcode-ui-' + this.model.get( 'attr' ) + '-' + this.model.cid,
 		}, this.model.toJSON() );
 
+		// Handle legacy custom meta.
+		// Can be removed in 0.4.
+		if ( data.placeholder ) {
+			data.meta.placeholder = data.placeholder;
+			delete data.placeholder;
+		}
```
Comment (danielbachhuber) [inline js/build/shortcode-ui.js:624]: Hm, this shouldn't be included.

File: js/build/shortcode-ui.js [hunk 2]
```diff
@@ -206,12 +206,18 @@ Shortcode = Backbone.Model.extend({
 
 		} );
 
+        console.log( attrs.length );
```
Comment (danielbachhuber) [inline js/build/shortcode-ui.js]: Please remove `console.log()`

File: js/build/shortcode-ui.js [hunk 3]
```diff
@@ -206,12 +206,18 @@ Shortcode = Backbone.Model.extend({
...
+
 		if ( this.get( 'inner_content' ) ) {
 			content = this.get( 'inner_content' ).get( 'value' );
 		}
 
 		template = "[{{ shortcode }} {{ attributes }}]"
 
+        if( attrs.length < 1 ) {
+            template = "[{{ shortcode }}]"
+        }
```
Comment (danielbachhuber) [inline js/build/shortcode-ui.js:217]: Looks like indentation is off a bit here. Also, can you make this an `if () else {}` statement?

File: js/build/shortcode-ui.js (hunk 1 above)
Comment (Kuzmanov) [inline js/build/shortcode-ui.js:624]: I've no idea why this code has been added while compiling. I've removed in the new commit and it appears again.

Comment (danielbachhuber) [inline js/build/shortcode-ui.js:624]: Can you try creating a fresh PR from master?

**Comment by danielbachhuber** [comment]
This will be fixed by #334

**Pull request closed without merging**